ManatiCR/manatime
# 50: Timesheet view.

**Steps to test:**
- Create some projects and add some tasks.
- Create a timesheet node, add some time-entries to the test.
- Remember the timesheet must be unpublished if you want to see something in /timesheet.
- Check if the button 'add row' run successfully.

[issue link](https://github.com/ManatiCR/manatime/issues/50)
